__tests__/index.test.js

```diff
@@ -54,3 +54,41 @@ describe('GET /api endpoints', () => {
 			});
 	});
 });
+
+describe('GET /api/articles/:article_id', () => {
+	test('GET:200 sends a single article to the client', () => {
+		return request(app)
+			.get('/api/articles/1')
+			.expect(200)
+			.then(({ body }) => {
+				for (const key in body) {
```

Comment:
Why would you need a for... in loop here? You should have one property in the body and an article object on the key of article. Please remove that unnecessary loop.